### ISSUE TITLE
Luci: Keyboard navigation in the main navbar doesn't unfold menus, no clear way to switch between menus and their items

### ISSUE THREAD

**Issue by nekohayo** [openwrt/luci#7901]
### Is there an existing issue for this?

- [x] I have searched among all existing issues (including closed issues)

### Actual behaviour

When you load the Luci GUI and try to use it only with the keyboard (because the computer you're using doesn't have a mouse, or some other accessibility-related reason), it seems to behave in unexpected ways, especially if you are a sighted user.

* It doesn't let you easily switch between top level menus beyond the 1st one when pressing the `Tab` (and `Shift+Tab`) keys
* It doesn't let you open/(un)fold the menus to see their contents

### Expected behaviour

I'm not a web accessibility expert; that said, my hunch is that Luci's GUI should be entirely keyboard-navigatable and that it should let me:

* Navigate between each top-level menus with `Tab` (forwards) and `Shift+Tab` (backwards)
* Open up (or close) those menus somehow (maybe with `Spacebar`?)
* Navigate between the items of those menus (maybe with the up/down arrows, I suppose?)
* Activate them with the `Enter` key

Potentially relevant: https://www.w3.org/WAI/tutorials/menus/flyout/

### Steps to reproduce

1. Load the Luci web interface in any browser
2. Press `Tab` once: the router's hostname is highlighted
3. Press `Tab` again: the 1st toplevel menu ("Status") is highlighted
4. Press `Spacebar`: the menu doesn't open (but the page somehow scrolls)
5. Pressing `Tab` a 3rd time: the highlight disappears, it _seems_ impossible to go directly to the next toplevel menu (but in fact, what happens behind the scenes is that it invisibly cycles through the items within the current "Status" menu; I'm not sure it should be doing that on `Tab`, when the menu is not opened)

### Additional Information

```shell
* Model	Netgear: R6850
* Architecture: MediaTek MT7621 ver:1 eco:3
* Target Platform: ramips/mt7621
* Firmware Version: OpenWrt 23.05.5 r24106-10cc5fcd00 / LuCI openwrt-23.05 branch git-24.264.56413-c7a3562
* Kernel Version: 5.15.167

I have tested this on Firefox and on Edge.
```

### What browsers do you see the problem on?

Firefox

### Relevant log output

```shell

```
